### PR TITLE
hotfix: dayinfo now shows holidays

### PR DESCRIPTION
**Pull request by Adman**
Signed-off-by: Adrian Matejov <a.matejov@centrum.sk>

### CODE REVIEW

**Comment by Adman** [comment]
@mrshu 
cc @pepol

File: dayinfo.py [hunk 1]
```diff
@@ -78,11 +78,12 @@ def dayinfo(event):
     args = event.args
     if len(args) == 1:
         try:
-            today = datetime.strptime(event.args[0], DATE_FORMAT)
+            today_s = datetime.strptime(event.args[0], DATE_FORMAT)
+            today = date(today_s.year, today_s.month, today_s.day)
```
Comment (mrshu) [inline dayinfo.py]: @Adman why does this have to be in a try except block?

File: dayinfo.py [hunk 2]
```diff
@@ -78,11 +78,12 @@ def dayinfo(event):
...
         except ValueError:
             return 'Not a valid date'
     else:
-        today = datetime.today()
+        today = date.today()
```
Comment (mrshu) [inline dayinfo.py]: @Adman wouldn't it be better to have this just below like 78? It does not seem that the else part is really necessary.

**Comment by mrshu** [comment]
@Adman generally looks good, just a couple of comments.

**Comment by Adman** [comment]
@mrshu should be fixed

**Comment by mrshu** [comment]
@Adman this looks pretty nice, thanks!

**Pull request merged**